grapple/types/redirects.py

```diff
@@ -1,37 +1,61 @@
+import json
+
+from typing import Optional
+
 import graphene
 
-from django.conf import settings
 from wagtail.contrib.redirects.models import Redirect
+from wagtail.models import Page, Site
 
 from .pages import get_page_interface
 
 
-class RedirectType(graphene.ObjectType):
+class RedirectObjectType(graphene.ObjectType):
```

Comment:
So that developers can more clearly see what `Site` this redirect applies to, can you add a `site` field to this?

Reply:
Addressed in 33561505891fe6ec7d6a5901f9301af1d26aaad5